pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -199,14 +205,30 @@ func (o *capability) loop() {
 		select {
 		case <-ctx.Done():
 			return
-		case r := <-o.newExpiryWorkerCh:
+		case r := <-o.newTimerCh:
 			o.wg.Add(1)
-			go o.expiryWorker(ctx, r)
+			go o.expiryTimer(ctx, r)
+		case resp := <-o.transmitCh:
+			o.handleTransmitMsg(ctx, resp)
 		}
 	}
 }
 
-func (o *capability) expiryWorker(ctx context.Context, r *request) {
+func (o *capability) handleTransmitMsg(ctx context.Context, resp *response) {
+	req, wasPresent := o.store.evict(ctx, resp.WorkflowExecutionID)
+	if !wasPresent {
+		o.lggr.Debugf("failed to clean up request with id %s", resp.WorkflowExecutionID)
```

Comment:
We will see this log for every single request, right? IIUC we never cancel expiry timers early.